src/lib.rs

```diff
@@ -1247,6 +1259,24 @@ unsafe impl<'a> alloc::Alloc for &'a Bump {
     }
 }
 
+#[cfg(feature = "allocator_api")]
+unsafe impl<'a> Allocator for &'a Bump {
```

Comment:
Can you also implement `grow` and `shrink`? See the existing `realloc` method for details (and maybe make `realloc` branch and call these new methods, so there isn't code duplication)

Reply:
Done